test/rest_request_template/travis_correct_multiple_topic_to_one_table_snowpipe.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "SNOWFLAKE_CONNECTOR_NAME",
+  "config": {
+    "connector.class": "com.snowflake.kafka.connector.SnowflakeSinkConnector",
+    "snowflake.role.name": "TESTROLE_KAFKA",
+    "topics": "SNOWFLAKE_TEST_TOPIC",
+    "snowflake.topic2table.map": "NUM_TOPICS<3>NUM_TOPICS",
```

Comment:
could you explain what is `NUM_TOPICS<3>NUM_TOPICS`? My understanding is that we need to provide a map here, so something like `topic1:table,topic2:table`

Reply:
Enclosed in <> is the number of topics to be generated. I added this to make the number of topics both configurable and compatible to the old behavior of the test script.

Reply:
It is basically equivalent to `topic1:table,topic2:table,topic3:table`, except that the name of the table and topics are not hard-coded.